sympy/polys/matrices/tests/test_lll.py

```diff
@@ -0,0 +1,123 @@
+from sympy.core.numbers import Rational
+from sympy.polys.domains import ZZ, QQ
+from sympy.polys.matrices.ddm import DDM
+from sympy.polys.matrices.domainmatrix import DomainMatrix
+from sympy.polys.matrices.exceptions import DMRankError
+from sympy.polys.matrices.lll import ddm_lll
+from sympy.testing.pytest import raises
+
+normal_test_data = [
+    (
+        DDM(
+            [
+                [ZZ(1), ZZ(0), ZZ(0), ZZ(0), ZZ(-20160)],
```

Comment:
A recent PR added the `DM` function which can make this sort of thing a bit easier:
```python
In [1]: from sympy.polys.matrices import DM

In [2]: DM([[1, 2], [3, 4]], ZZ)
Out[2]: DomainMatrix([[1, 2], [3, 4]], (2, 2), ZZ)

In [3]: DM([[1, 2], [3, 4]], ZZ).rep
Out[3]: DDM([[mpz(1), mpz(2)], [mpz(3), mpz(4)]], (2, 2), ZZ)
```
Unlike the basic constructor it converts all of the inputs to the domain so it isn't necessary to use `ZZ(1)` rather than `1`.

Reply:
Ok I am changing it to use the DM constructor